xml/decoders/soundtraxx/TSU2CVLightEffect.xml

```diff
@@ -40,4 +40,10 @@
       <xi:include href="http://jmri.org/xml/decoders/soundtraxx/TSU2enumEffect.xml"/>
       <label>FX6 Effect Selection</label>
     </variable>
+    <variable item="FX7 Effect Selection" CV="55" mask="XXXVVVVV" default="15" minOut="7" tooltip="Determines the second effect associated with the FX7 light output">
+      <xi:include href="http://jmri.org/xml/decoders/soundtraxx/TSU2enumEffect.xml"/>
+    </variable>
```

Comment:
The existing FX6 has an "FX6 Effect Selection" label.  Should FX7 and FX8 have that also?

Reply:
Not really, the first 6 have item names that are for TSU1 decoders and the pane uses the TSU1 names, with labels to give them TSU2 displayed names.  The new outputs are TSU2 only so the pane name matches the item name which is the display name desired.  Can add a label but it would be exactly the same as the item.